src/Stratis.Bitcoin.Features.Wallet/WalletRPCController.cs

```diff
@@ -29,13 +41,51 @@ public uint256 SendToAddress(BitcoinAddress bitcoinAddress, Money amount)
             return uint256.Zero;
         }
 
+        /// <summary>
+        /// Broadcasts a raw transaction from hex to local node and network.
+        /// </summary>
+        /// <param name="hex">Raw transaction in hex.</param>
+        /// <returns>The transaction hash.</returns>
+        [ActionName("sendrawtransaction")]
+        [ActionDescription("Submits raw transaction (serialized, hex-encoded) to local node and network.")]
+        public async Task<uint256> SendTransactionAsync(string hex)
+        {
+            this.logger.LogTrace("({0}:'{1}')", nameof(hex), hex);
+
+            Transaction transaction = this.fullNode.Network.CreateTransaction(hex);
+            await this.broadcasterManager.BroadcastTransactionAsync(transaction);
+
+            uint256 hash = transaction.GetHash();
+
+            this.logger.LogTrace("(-):'{0}'", hash);
+            return hash;
+        }
+             
+        /// <summary>
+        /// RPC method that gets a new address for receiving payments.
+        /// Uses the first wallet and account.
+        /// </summary>
+        /// <returns>The new address.</returns>
+        [ActionName("getnewaddress")]
+        [ActionDescription("Returns a new wallet address for receiving payments.")]
+        public NewAddressModel GetNewAddress()
```

Comment:
`unusedaddress`/`unusedaddresses`. Note that this probably means you can remove the `NewAddressModel` entirely.

Reply:
Unfortunately not, RPC has some funky serialization logic so need a separate model in this case.